docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -119,6 +122,31 @@ field (``bdb-certs.bdb-user``), i.e.
    emailAddress=dev@bigchaindb.com,CN=test-bdb-ssl,OU=BigchainDB-Instance,O=BigchainDB GmbH,L=Berlin,ST=Berlin,C=DE
 
 
+threescale-credentials.*
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete the ``threescale-credentials`` Secret if you're not using 3scale.
```

Comment:
`or leave it blank`, which is much simpler than asking to delete a formatted file as yaml.